libPhoneNumber/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,120 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class NBPhoneNumber;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBPhoneNumberOfflineGeocoder : NSObject
+
+/**
+ * Returns a text description for the given phone number, in the language provided. The
+ * description might consist of the name of the country where the phone number is from, or the
+ * name of the geographical area the phone number is from if more detailed information is
+ * available.
+ *
+ * This method assumes the validity of the number passed in has already been checked, and that
+ * the number is suitable for geocoding. We consider fixed-line and mobile numbers possible
+ * candidates for geocoding.
+ *
+ * @param phoneNumber  a valid phone number for which we want to get a text description
+ * @param languageCode  the language code for which the description should be written
+ * @return  a text description for the given language code for the given phone number, or an
+ *     empty string if the number could come from multiple countries, or the country code is
```

Comment:
nil.

Also, double check the comments to make sure it matches the behavior.

Reply:
done.